lxd/storage_volumes_snapshot.go

```diff
@@ -733,13 +730,11 @@ func autoCreateCustomVolumeSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
 			if len(nodes) > 1 {
 				seed := fmt.Sprintf("%d", v.ID)
 
-				hash := fnv.New64a()
-				_, err = io.WriteString(hash, seed)
+				r, err := util.GetStableRandomGenerator(seed)
 				if err != nil {
 					continue
 				}
 
-				r := rand.New(rand.NewSource(int64(hash.Sum64())))
 				selectedNodeID := availableNodeIDs[r.Int63n(int64(len(availableNodeIDs)))]
```

Comment:
@tomponline We should use `GetStableRandomInt64FromList` here as well.

Reply:
@monstermunchkin ah yes, indeed, fixed.